tests/functional/firefox/test_notes.py

```diff
@@ -0,0 +1,113 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.firefox.accounts import FirefoxNotesPage
+
+
+@pytest.mark.nondestructive
+def test_firefox_desktop_release_page(base_url, selenium):
+    #firefox/65.0/releasenotes/
+    page = FirefoxNotesPage(selenium, base_url).open()
+    assert page.version is '65.0'
+    assert page.channel is 'Firefox Release'
+    assert page.gtm_id is '/firefox/releasenotes/'
+    assert page.download_button_text is 'Download Firefox'
+    assert page.download_button_link.contains('firefox-stub')
```

Comment:
Something like this will only pass on Windows, so I'm not sure if this will work? 

We already have good unit tests for dl button URLs in Python. As far as a functional test goes, I'd try and concentrate on things that the visitor experiences on the page:

- Is the download button visible?
- Is the right download button channel shown for the type of release notes page?

I think asserting the two things above would probably be enough? ^

Reply:
By looking at the first child I'm always looking at the windows button :)

> Is the right download button channel shown for the type of release notes page?

This is indeed what I really want to test, how would you suggest I test it?

Reply:
> This is indeed what I really want to test, how would you suggest I test it?

I could look for the right CSS class in the download button.

Reply:
I don't see a CSS class that corresponds to channel on the div, ul, li, or a 🙁

Reply:
Ah, there isn't one on release but there is one for beta and nightly.

Reply:
You can also give each button a unique Id to make targeting easier